mozci/push.py

```diff
@@ -411,10 +403,38 @@ def group_summaries(self):
         for future in concurrent.futures.as_completed(future_to_task):
             task = future_to_task[future]
             for group in future.result():
-                groups[group].append(task)
+                yield task, group
+
+    @memoized_property
+    def group_config_summaries(self):
+        """All group summaries, on given configurations, combining retriggers.
+
+        Returns:
+            dict: A dictionary of the form {(<configuration>, <group>): [<GroupSummary>]}.
+        """
+        config_groups = defaultdict(list)
```

Comment:
I wish I could find a better term to define a (<configuration>, <group>) couple...

Reply:
Wouldn't that be a `runnable`?

Reply:
Yeah but by `runnable` we basically mean "a possible grouping of task", so "label", "group" and this new one are all possible "runnables". I'd like a name which is not exactly "group" (as we still need that until bugbug uses this "configgroup" only).

Reply:
Maybe we should just expand the meaning of `test_platform` to include runtime information such as `fission`. Another alternative might be `test_context`, though `context` is also used frequently.